app/ui-react/syndesis/craco.config.js

```diff
@@ -15,13 +24,17 @@ module.exports = function({ env, paths }) {
           ],
         },
       }),
-    },
-    jest: {
-      configure: (jestConfig, { env, paths, resolve, rootDir }) => ({
-        ...jestConfig,
-
-        moduleFileExtensions: ['ts', 'tsx', 'js'],
-      }),
```

Comment:
I don't think we want to remove this block.

Reply:
restored